analysers/analyser_osmosis_relation_route_access.py

```diff
@@ -104,5 +104,5 @@ def analyser_osmosis_full(self):
 
     def analyser_osmosis_diff(self):
         for route_type, access in self.map.items():
-            self.run(sql10.format('', 'touched_', route_type, access['acces_tag'], access['no_acces'], access['highway_overide'], access['highway_overide_access']), self.callback10(access['class']))
-            self.run(sql10.format('touched_', 'not_touched_', route_type, access['acces_tag'], access['no_acces'], access['highway_overide'], access['highway_overide_access']), self.callback10(access['class']))
+            self.run(sql10.format('', 'touched_', route_type, access['access_tag'], access['no_access'], access['highway_overide'], access['highway_overide_access']), self.callback10(access['class']))
+            self.run(sql10.format('touched_', 'not_touched_', route_type, access['access_tag'], access['no_access'], access['highway_overide'], access['highway_overide_access']), self.callback10(access['class']))
```

Comment:
The issue should be in this line right ???
But in the error message the names don't match

Reply:
https://github.com/osm-fr/osmose-backend/pull/761/files/6c2de4442a81670b76858f422cd7a2e73787c2b6#diff-b631ed634ba0b61cf2bcc5121104356fR103